config/vendor-api-0.4.0.yml

```diff
@@ -125,6 +125,8 @@ paths:
           $ref: "#/components/responses/Unauthorized"
         "404":
           $ref: "#/components/responses/NotFound"
+        "422":
+          $ref: "#/components/responses/MetadataMissing"
```

Comment:
Perhaps I would make the `422` a generic error for when not enough/incorrect data is sent, not just metadata related.

Reply:
:+1: This will be useful in handling offer-conditions validations too

Reply:
renamed to 
`$ref: "#/components/responses/BadRequestBody"`
BUT
I've just noticed a similar 422 one:
`"422":
          $ref: "#/components/responses/ParameterMissing"
`
should we merge them?

Reply:
No — these are different errors, I think. ParameterMissing tells us a URL param is missing. BadRequestBody tells us the request body was invalid.